WikiWatershed/model-my-watershed
# Add analyze related layers dropdown view

## Overview

Style the water quality related layers:

![image](https://cloud.githubusercontent.com/assets/7633670/25347650/a237bdc2-28e9-11e7-9e5c-348cccbd1b31.png)

Wireframe
![image](https://cloud.githubusercontent.com/assets/7633670/25347679/b93600b0-28e9-11e7-8280-899eb42211ad.png)

(See more wireframes in connecting issue)

Connects #1834 

### Demo

![3y2ujiisyc](https://cloud.githubusercontent.com/assets/7633670/25348533/8e7679e2-28ec-11e7-97b6-563937f14888.gif)



### Notes

~Whether we're going to display the short display or full display once you select a layer is pending https://github.com/WikiWatershed/model-my-watershed/issues/1834#issuecomment-296711118~
added 644bc89 to show full display name


## Testing Instructions

 * Pull and `./scripts/bundle.sh` (if you already have mmw running, you many need to `vagrant ssh app -c 'sudo restart mmw-app'` to get the new short display names)
 * Check that 
    * On the water quality analyze tab for an AOI inside the DRB, there's a dropdown "3 related layers" the toggles the catchment layers both in the analyze sidebar and layerpicker (and on the map)
    * For an AOI outside the DRB, there should a be a "no related layers available" message and no dropdown
    * All other analyze tabs' related layers should work as before
